packages/chat-adapter/src/index.ts

```
@@ -0,0 +1,717 @@
+      participantIDs: [participantID],
+      type: "single",
+    });
+    return this.encodeThreadId({ accountID, chatID: created.chatID });
+  }
+
+  encodeThreadId(platformData: BeeperDesktopAPIThreadID): string {
+    return `beeperDesktopAPI:${encodeURIComponent(platformData.accountID)}:${encodeURIComponent(platformData.chatID)}`;
+  }
+
+  decodeThreadId(threadID: string): BeeperDesktopAPIThreadID {
+    const [prefix, accountID, chatID] = threadID.split(":");
```

ESLint

[error] Replace `":"` with `':'`

(prettier/prettier)